package.json

```diff
@@ -9,6 +9,7 @@
     "start:firefox": "concurrently 'npm run webpack:dev -- --watch' 'web-ext run --config=.web-ext-config.js'",
     "build:prod": "npm run webpack:prod && web-ext build --config=.web-ext-config.js",
     "test": "jest --coverage && npm run webpack:prod && web-ext lint -s dist",
+    "test:local": "jest",
```

Comment:
Also, no need for this.

Reply:
it looks like our other test script is used by ci or something, it's way too much for just running tests.

Reply:
i would keep this

Reply:
Yes, it's ok.